Web_Development/Javascript/Topics/5.Syntax/2.Operators/ComparisonOperator.md

```diff
@@ -0,0 +1,64 @@
+# Comparison Operators
+[Back to Operators in Javascript](README.md#operators-in-javascript)
+
+Comparison operators are used to compare values and returns boolean value (true or false).
+
+#### 1. Equal Operator (==)
+It will return true if both values are equal otherwise false.
+
+```js
+console.log(5==5); // true
+console.log(4==3); // false
+```
+
+#### 2. Not Equal Operator (!=)
+It will return true if both values are not equal otherwise false.
+
+```js
+console.log(5!=5); // false
+console.log(4!=3); // true
+```
+
+#### 3. Strict Equal Operator (===)
+It will check values and also check the type of the values.
+
+```js
+console.log(5===5); // true
+console.log('5'===5); // false // left value is string and right value is number
+console.log(4===3); // false
+```
+
```

Comment:
Add `Strict Not Equal To Operator (!==)` and write about it also.
Also talk about how this should be the mainstream practice as in this example `0 !== '0'` results in **false** while **not equal to operator (!=)** results in **true** in the same expression.